src/App.js

```diff
@@ -12,39 +13,121 @@ class App extends React.Component {
       // guessedLetters stores all letters a user has guessed so far
       guessedLetters: [],
       // Insert num guesses left state here
+      numGuessLeft: 10,
       // Insert form input state here
+      currInput: "",
+      wordDisplay: [],
+      hasPlayerGuessedCorrectly: true,
     };
   }
 
   generateWordDisplay = () => {
-    const wordDisplay = [];
+    let { currWord, guessedLetters, wordDisplay } = this.state;
+
     // for...of is a string and array iterator that does not use index
-    for (let letter of this.state.currWord) {
-      if (this.state.guessedLetters.includes(letter)) {
+
+    for (let letter of currWord) {
+      console.log(currWord);
+      if (guessedLetters.includes(letter)) {
         wordDisplay.push(letter);
       } else {
         wordDisplay.push("_");
       }
     }
-    return wordDisplay.toString();
+    // display word and make text in textbox empty
+    this.setState({
+      wordDisplay: wordDisplay,
+      currInput: "",
+    });
   };
 
   // Insert form callback functions handleChange and handleSubmit here
+  handleChange = (event) => {
+    this.setState({
+      currInput: event.target.value,
+    });
+  };
+
+  handleSubmit = (event) => {
+    let { currInput, guessedLetters, currWord } = this.state;
+    event.preventDefault();
+    if (currInput === "" || currInput.length > 1) {
+      alert("Please input a letter");
+      this.setState({
```

Comment:
```suggestion
      return this.setState({
```

return here, and you can remove the if else spaghetti below